routes/badges.js

```diff
@@ -302,7 +299,7 @@ module.exports = function (env) {
         if (err) {
           var errorString = err.message;
           if (err.code === 404) {
-            errorString = 'The code "' + req.body.claimcode + '" could not be found';
+            errorString = 'The code \'' + req.body.claimcode + '\' could not be found';
```

Comment:
Ah, I know what happened.

It was initially `'The code "'`

Which then became `"The code \""` after my first quote change.

Then a lazy find replace turned it into `'The code \''`